src/seedu/addressbook/commands/Command.java

```diff
@@ -46,6 +48,12 @@ public CommandResult execute(){
     /**
      * Supplies the data the command will operate on.
      */
+    public void setData(AddressBook addressBook, List<? extends ReadOnlyPerson> relevantPersons, StorageFile storage) {
```

Comment:
List<? extends ReadOnlyPerson>, what does it mean "? before that extends"?

Reply:
This is actually an example of wildcard generics! You might want to read more on it here: http://tutorials.jenkov.com/java-generics/wildcards.html